lib/node/http.js

```diff
@@ -255,6 +259,23 @@ class HTTP extends Server {
       res.json(200, block.getJSON(this.network, view, height, depth));
     });
 
+    // Filters by hash/height
+    this.get('/filter/:block', async (req, res) => {
+      const valid = Validator.fromRequest(req);
+      const hash = valid.uintrhash('block');
```

Comment:
This will be a string currently, and needs to be a `Buffer`. It can use `uintbrhash` instead of `uintrhash` will give the `Buffer` for the hash.